pubspec.yaml

```diff
@@ -1,11 +1,11 @@
 name: flutter_web_auth
 description: Flutter plugin for authenticating a user with a web service.
-version: 0.2.4
+version: 1.0.0
 author: Linus Unnebäck <linus@folkdatorn.se>
 homepage: https://github.com/LinusU/flutter_web_auth
 
 environment:
-  sdk: ">=2.1.0 <3.0.0"
+  sdk: ">=2.12.0-29.10.beta <3.0.0"
```

Comment:
Where did you find this value? Does it have to be beta? 🤔

Reply:
It is from migration guide:
```
Set the lower SDK constraint to the beta version of 2.12 that you used to test the migration, and the upper SDK constraint to <3.0.0. For example, if you’re using 2.12.0-29.10.beta, then your constraints should look like this:
```

Reply:
This should be set to `2.12.0` imo

Reply:
This constrain includes 2.12.0 as well. Support For null-safety has been added in 2.12.0-29.10.beta thats why Official migration guide recommend to but this value, everything after this will work either.

Reply:
While that was true before the official release of Flutter 2, it is now recommended to use `2.12.0` as a lower bound: https://dart.dev/null-safety/migration-guide#migrating-by-hand

Thanks for the quick response and change, though. Highly appreciated :)